src/frontend/main/CityTables.tsx

```diff
@@ -170,73 +175,38 @@ const CityTables = ({ data, selectedCity, isEmbed }: CityTablesProps) => {
               <CityHeading>{item.city}</CityHeading>
               {formattedData.responsesTotal != null ? (
                 <>
-                  <BoldText>Vastauksia yhteensä {formattedData.responsesTotal}</BoldText>
-                  <ItalicText>{formattedData.percentageOfPopulation} % väkiluvusta</ItalicText>
+                  <BoldText>
+                    {t('main:totalResponses')} {formattedData.responsesTotal}
+                  </BoldText>
+                  <ItalicText>
+                    {formattedData.percentageOfPopulation} % {t('main:ofPopulation')}
```

Comment:
There should be a space before % in Finnish but not in English. There were a couple places with the same problem, I'll try to find them in the code... So many weird details that have to be checked when you are "just" doing translations 😅

Reply:
Ahh, I thought the space was just global convention :sweat_smile: 
Going to add a translation for percentage number